.travis.yml

```diff
@@ -1,20 +1,64 @@
-dist: bionic
+# TravisCI configuration for mautic/mautic
+
 language: php
+os:
+  - linux
+dist: bionic
 
 services:
   - mysql
 
-notifications:
-    webhooks: https://www.travisbuddy.com/
-    on_success: never
-
 php:
   - 7.2
   - 7.3
 
 cache:
   directories:
-    - $HOME/.composer/cache
+    - "${HOME}/.composer/cache"
+
+env:
+  global:
+    # Set to test environment for Symfony's commands in post install commands
+    - SYMFONY_ENV="test"
+
+jobs:
+  include:
+    - stage: test
+      name: "MariaDB, PHP 7.2"
+      php: 7.2
+      env: DB="mariadb"
+      addons:
+        mariadb: 10.2
+    -
+      name: "MariaDB, PHP 7.3"
+      php: 7.3
+      env: DB="mariadb"
```

Comment:
Could we maybe add an environment variable `RUN_COVERAGE` or similar, so that code coverage runs on one of the PHP 7.3 runs? The extra stage "coverage" below adds ~20 mins to the overall build time, as Travis can run a maximum of 4 jobs simultaneously 🙈

Reply:
I'm trying to process this contradiction as this PR's sole purpose is to eliminate conditional :)

Reply:
What about something like `PHPUNIT_ARGS` with value `--coverage-clover=coverage.xml`? Then we don't have to add conditionals but rather just do `composer test -- $PHPUNIT_ARGS` (assuming this will work)?

Reply:
Done.

![kép](https://user-images.githubusercontent.com/952007/96503560-bf89d980-1253-11eb-9c1d-03bea95adfea.png)